src/pages/login.js

```diff
@@ -35,7 +45,27 @@ class Login extends Component {
           autoCapitalize={'none'}
           secureTextEntry={true}
         />
-        <Button primary text="Login" />
+        <View style={core_styles.facebookBtn}>
```

Comment:
maybe `facebookBtn` is too specific to be in core_styles, We could have it in this same file or in another file for the login page